deploy/wip-catalog/templates/apiserver.yaml

```diff
@@ -37,11 +37,18 @@ metadata:
   name: apiserver
 spec:
 {{ if .Values.debug }}
+{{ if .Values.nodePort }}
+  type: NodePort
+{{ else }}
   type: LoadBalancer
 {{ end }}
+{{ end }}
```

Comment:
Proof of the above, btw:

```
$ kubectl describe svc apiserver -n catalog | grep Type:
Type:			LoadBalancer

$ kubectl get svc apiserver -n catalog
NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)          AGE
apiserver   10.0.0.118   <pending>     6443:32441/TCP   5m

$ telnet $(minikube ip) 32441
Trying 192.168.99.112...
Connected to 192.168.99.112.
Escape character is '^]'.
```

P.S.-- I telnetted instead of curling to demonstrate the connection works only because I wasn't about to go messing around with certs.